docs/support/tutorials/gis/eo_guide.md

```diff
@@ -39,221 +54,166 @@ What to consider when chosing data:
 
-[**Sen2mosaic**](../../../apps/sen2cor.md)
 
-* A command-line tool to download, preprocess and mosaic Sentinel-2 data.
+## Alternative processing services
+Below is a list of alternative EO processing services that might be useful, when a lot of data is required and downloading it all to CSC might not be feasible. All of them include the main open datasets: Sentinel, Landsat, MODIS etc.
 
-[**SNAP**](../../../apps/snap.md) - ESA Sentinel Application Platform
+**[Google Earth Engine](https://earthengine.google.com/)** is a processing platform, which requires registration, but is free of charge for research users. It can be accessed via browser and has worldwide analysis ready data available (<a href="https://developers.google.com/earth-engine/datasets/catalog/" aria-label="GEE data catalogue">browse the catalogue</a>). In general, JavaScript is used on the platform, but also  <a href="https://developers.google.com/earth-engine/guides/python_install" aria-label="Python on GEE">Python</a>  and  <a href="https://github.com/r-spatial/rgee" aria-label="R on GEE">R</a> support exists. Check out [GEE's tutorials](https://developers.google.com/earth-engine/tutorials). Note that Google Cloud Storage might be needed to export large datasets.
 
-* Tool for processing of Sentinel data (+ support for other data sources) 
-* GUI, CLI (Graph Processing Tool, GPT) and Python interfaces (snappy and snapista (snapista currently not available on Puhti))
-* [SNAP GPT example for Puhti](https://github.com/csc-training/geocomputing/tree/master/snap).
+**[Microsoft planetary computer](https://planetarycomputer.microsoft.com)**. For computing it offers JupyterHub together with Dask Gateway, both CPUs and GPUs are available. It is currently available in preview.
 
-### Machine Learning with EO data
+ **[Data and Information Access Services (DIAS)](https://www.copernicus.eu/en/access-data/dias)** cloud based Virtual Machines (VMs), dedicated baremetal servers, containers, operating system and software images. Specialized in EO, user support available. Commercial services. 
+ 
+[**Sentinelhub**](https://www.sentinel-hub.com/explore/). Several different APIs. Commercial service.
 
-One example of the advanced usage of EO data is for machine learning. If you are interested in the topic, you can find a lot of examples using EO and other geospatial data for machine learning in our [Machine learning with spatial data course exercises]( https://github.com/csc-training/GeoML) on Github. 
+**Commercial clouds**: Amazon, Google Cloud and Microsoft Azure, all provide virtual machines and other processing services, all of them have some local data, see links above. No EO support.
```

Comment:
```suggestion
**Commercial clouds**: Amazon, Google Cloud and Microsoft Azure, all provide virtual machines and other processing services, all of them have some local data, see links above. 
```

Reply:
I don't think that "thinking that they do not offer it" is enough reason to put it as info here.